how-to-trust-fiddler-root-certificate.md

```diff
@@ -0,0 +1,39 @@
+---
+title: How to trust the Fiddler Root certificate
```

Comment:
Capitalization needs fixing. 

You can use the following hefty tool: http://titlecapitalization.com/# Make sure to select the Chicago Manual of Style option.